packages/blockchain-link/src/workers/blockbook/index.ts

```diff
@@ -256,6 +352,13 @@ const subscribeBlock = async () => {
     return socket.subscribeBlock();
 };
 
+const subscribeFiatRates = async (currency?: string) => {
+    const socket = await connect();
+    common.addSubscription('fiatRates');
+    socket.on('fiatRates', onNewFiatRates);
```

Comment:
set listener only if !common.addSubscription('fiatRates') - like its done in `subscribeAddresses`, otherwise calling `subscribeFiatRates` multiple times will fire up onNewFiatRates multiple times